youtube-client/src/interfaces.ts

```diff
@@ -0,0 +1,36 @@
+export interface Video {
+  kind: string;
+  etag: string;
+  id: {
+    kind: string;
+    videoId: string;
+  };
+
+  snippet: {
+    title: string;
+    channelId: string;
+    channelTitle: string;
+    description: string;
+    liveBroadcastContent: string;
+    publishTime: string;
+    publishedAt: string;
+
+    thumbnails: {
```

Comment:
It could be beneficial in large types to create small one separately, for example snippet or thumbnails and then combine them